Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityController.cs

```diff
@@ -237,6 +238,40 @@ private void UpdateMenuData(InteractionSourceState interactionSourceState, Mixed
             }
         }
 
+        // Velocity internal states
+        private float deltaTimeStart;
+        private readonly int velocityUpdateInterval = 6;
+        private int frameOn = 0;
+        private Vector3[] velocityPositionsCache = new Vector3[6];
+        private Vector3 velocityPositionsSum = Vector3.zero;
+
+        protected void UpdateVelocity()
+        {
+            if (frameOn < velocityUpdateInterval)
```

Comment:
I'm not sure if we provide this data for hands, but for motion controllers [velocity](https://docs.unity3d.com/ScriptReference/XR.WSA.Input.InteractionSourcePose.TryGetVelocity.html) and [angular velocity](https://docs.unity3d.com/ScriptReference/XR.WSA.Input.InteractionSourcePose.TryGetAngularVelocity.html) is provided by the platform. Do we want to use those values instead of calculating our own?

Reply:
+1 one to that

Reply:
+1 to using the platform-supplied velocity where available, i would assume it's more accurate.

Reply:
That's cool, I didn't know about this one. It works for both MR controllers and for hands.